packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -96,6 +104,129 @@ export async function generateArmTemplate(ctx: SolutionContext): Promise<Result<
   return ok(undefined); // Nothing to return when success
 }
 
+export async function deployArmTemplates(ctx: SolutionContext) {
+  const azureInfraDir = path.join(ctx.root, baseFolder);
+  const orchestrationFilePath = path.join(
+    azureInfraDir,
+    templateFolder,
+    bicepOrchestrationFileName
+  );
+  const armTemplateJsonFilePath = path.join(azureInfraDir, templateFolder, armTemplateJsonFileName);
+
+  // TODO: update placeholder in parameter.template.json
+  const parameterTemplateFile = await fs.readFile(
+    path.join(azureInfraDir, parameterFolder, parameterTemplateFileName),
+    ConstantString.UTF8Encoding
+  );
+  const parameterDefaultFilePath = path.join(
+    azureInfraDir,
+    parameterFolder,
+    parameterDefaultFileName
+  );
+  const parametersObject: Json = {}; // TODO: update placeholder in parameter.template.json and get parameter json object
```

Comment:
Will these these TODOs be addressed in this commit?

Reply:
The file you see is outdated. Please look at the latest code. And the remaining TODO will not be done in this commit.